src/components/auth/SignIn.js

```diff
@@ -33,7 +35,7 @@ class SignIn extends React.Component {
             <input type="password" id="password" onChange={this.handleChange} />
           </div>
           <div className="input-field">
-            <button className="btn">Login</button>
+            <FooterButton>Login</FooterButton>
             <div className="red-text center" />
```

Comment:
Do we want to show it in red? I think we need to ask Patryk opinion on that. Maybe ask for the design of the banners.